### ISSUE TITLE
ValueError in PhraseMatcher.__call__

### ISSUE THREAD

**Issue by thomasopsomer** [explosion/spaCy#613]
Hello,

Using spacy PhraseMatcher now gives error due to deprecated use of acceptor keyword in Matcher. Here is a simple example:
```
import spacy
from spacy.matcher import PhraseMatcher

nlp = spacy.load("en")

MWEs = [u"machine-learning"]
vocab = nlp.tokenizer.vocab
matcher = PhraseMatcher(vocab=vocab, phrases=[], max_length=6)

for mwe in MWEs:
    phrase = nlp.tokenizer(mwe)
    matcher.add(phrase)

s = u"machine-learning is used for predictive analytics"
doc = nlp(s)
matcher(doc)
```
It results in:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "spacy/matcher.pyx", line 509, in spacy.matcher.PhraseMatcher.__call__ (spacy/matcher.cpp:11800)
  File "spacy/matcher.pyx", line 344, in spacy.matcher.Matcher.__call__ (spacy/matcher.cpp:8013)
ValueError: acceptor keyword argument to Matcher deprecated. Specify acceptor functions when you add patterns instead.
```

Looking at both class Matcher and PhraseMatcher, the `__call__` method of PhraseMatcher calls the `__call__` method of Matcher with acceptor=self.accept_match, which seems deprecated. 

## Your Environment

* **Operating System:** OSX 
* **Python Version Used:** 2.7.12
* **spaCy Version Used:**  1.2.0
* **Environment Information:**

**Comment by honnibal** [comment]
I think this should be fixed now, by #614. Please reopen if it's not.

**Comment by A-Jacobson** [comment]
Is this supposed to be fixed in 1.2.0? I'm experiencing the same error. I even tried the OP's exact code snippet and also received the same error.

**Comment by thomasopsomer** [comment]
Nope this fix isn't in the 1.2.0 release, the issue was opened just after it. I guess it will be in the next one. In the meantime If you clone the code on the master and install it manually, it works :)
